embassy-rp/Cargo.toml

```diff
@@ -60,8 +60,10 @@ chrono = { version = "0.4", default-features = false, optional = true }
 embedded-io = { version = "0.4.0", features = ["async"], optional = true }
 embedded-storage = { version = "0.3" }
 rand_core = "0.6.4"
+fixed = "1.23.1"
+typenum = { version = "1.14", default-features = false }
```

Comment:
`typenum` is unused

Reply:
`default-features = false` doesn't disable default features, it just doesn't enable them. If another crate depends on `typenum` without `default-features = false` then they get enabled.